menu/linuxday_at_2019.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 2019092901,
+  "url": "https://www.linuxday.at/vortraege/2019/calendar.ics",
+  "title": "LinuxDay.at Vortraege",
+  "id": "linuxday_at_2019",
```

Comment:
That doesn't seem to actually match your calname? I haven't bothered making the CI "clever" enough to check this because instead I just hope to eventually fix this id bug.

X-WR-CALNAME;VALUE=TEXT:LinuxDay.at Vortraege